utils/Consts.ts

```diff
@@ -59,23 +59,9 @@ after Capybara api: :ChocolaPOGGERS:`,
     worksAt: "Rabbit",
   },
   {
-    message: "best api cuz they do be pulling up in a quick efficient manner.",
-    name: "shinkuWRLD",
-    pfp: "/img/shinku.png",
-    worksAt: "ur moms house",
-  },
-  {
-    message:
-      "did you know that capypabaras eat their own poop in the morning? unique diet. protein rich. he pull up.",
-    name: "Ari",
-    pfp: "/img/ari.png",
-    worksAt: "Buildergroop Corporation",
-  },
```

Comment:
add these back

Reply:
to keep one of each deleted the ones being repeated

Reply:
ohh okay my b